hyperspyui/plugins/eelsdb.py

```diff
@@ -127,19 +128,23 @@ def default(self):
             self.window.setWindowTitle("EELSDB")
             vbox = QtWidgets.QVBoxLayout()
             self.window.setLayout(vbox)
-            view = QtWebEngineWidgets.QWebEngineView(self.ui)
+            view = QWebEngineView(self.ui)
             self.view = view
             vbox.addWidget(view)
             self.window.resize(self.view.sizeHint())
         # Load spectra browser
         browse_url = QtCore.QUrl("https://eelsdb.eu/spectra")
         self.load_blocking(self.view, browse_url)
         # Remove header/footer
-        frame = self.view.page().mainFrame()
-        frame.findFirstElement(".footer").removeFromDocument()
-        for el in frame.findAllElements(".navbar"):
-            el.removeFromDocument()
-        self.view.page().setLinkDelegationPolicy(
-                QtWebEngineWidgets.QWebEnginePage.DelegateAllLinks)
-        self.view.linkClicked.connect(self._on_link)
+        # frame = self.view.page().mainFrame()
+        # frame.findFirstElement(".footer").removeFromDocument()
+        # for el in frame.findAllElements(".navbar"):
+        #     el.removeFromDocument()
```

Comment:
I'd rather old code was deleted if no longer used.

Reply:
True, this is now done.